addons/knowledge/controllers/main.py

```diff
@@ -61,6 +61,24 @@ def redirect_to_article(self, article_id):
             'show_sidebar': bool(self._get_root_articles(limit=1))
         })
 
+    @http.route('/knowledge/article', type='http', auth='user')
+    def access_knowledge_home(self):
```

Comment:
Ordering: should be defined first, as this is a global route (globals, then article-specifc, then tools, ...)